cmd/monaco/dynatrace/dynatrace.go

```diff
@@ -211,14 +193,19 @@ func CreateEnvironmentClients(ctx context.Context, environments manifest.Environ
 	return clients, nil
 }
 
+// getDynatraceClassicURL transforms the platformURL to a classic URL either via string replacing or API call, depending on if the BuildSimpleClassicURL FF is enabled (default) or not
 func getDynatraceClassicURL(ctx context.Context, platformURL string, oauthCreds clientcredentials.Config) (string, error) {
 	if featureflags.BuildSimpleClassicURL.Enabled() {
 		if classicURL, ok := findSimpleClassicURL(ctx, platformURL); ok {
 			return classicURL, nil
 		}
 	}
 
-	client, err := clients.Factory().WithPlatformURL(platformURL).WithOAuthCredentials(oauthCreds).CreatePlatformClient(ctx)
+	factory := clients.Factory().WithPlatformURL(platformURL).WithOAuthCredentials(oauthCreds)
+	if supportarchive.IsEnabled(ctx) {
+		factory = factory.WithHTTPListener(&corerest.HTTPListener{Callback: trafficlogs.GetInstance().LogToFiles})
+	}
+	client, err := factory.CreatePlatformClient(ctx)
 	if err != nil {
 		return "", err
```

Comment:
Question: should we wrap this error like in line 99 above: `return fmt.Errorf("could not create client: %w", err)`?